test/unit/materialized_view_activity_test.sql

```diff
@@ -0,0 +1,72 @@
+set client_encoding = 'utf-8';
+set client_min_messages = warning;
+create extension if not exists pgtap;
+reset client_min_messages;
+
+begin;
+select plan(33);
+
+select has_materialized_view(
+    'ggircs_swrs', 'activity',
+    'should have materialized view activity'
+);
+
+select has_index
```

Comment:
Missing '(' after has_index
```suggestion
select has_index(
```